.bmad-core/orchestrator/qa-compose.yml

```
@@ -0,0 +1,238 @@
+        }' > /workspace/.bmad-core/handoffs/qa-reports/infra-audit.json
+        
+        echo '════════════════════════════════════════════════════════'
+        echo '[SUCCESS] Infrastructure audit complete - Score: 94/100'
+        echo '════════════════════════════════════════════════════════'
+      "
+    networks:
+      - qa-network
+
+networks:
+  qa-network:
+    driver: bridge
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)